model/scikit/dffml_model_scikit/scikit_models.py

```diff
@@ -241,8 +241,8 @@ def applicable_features(self, features):
         dffml_config_properties["predict"] = (
             Feature,
             field(
-                "field here for compability with other functions,no need to change",
-                default=DefFeature(name="Prediction", dtype="str", length=10),
+                "Name used as meaning of prediction",
+                default=DefFeature(name="cluster", dtype="str", length=10),
```

Comment:
I think this change was supposed to be only in #441? You can do them both in one PR, and close one of them, or remove this change from this PR.